### PR TITLE
Feautre/animation page

### PR DESCRIPTION
**Pull request by AlexAnarcho**
- Mock-up Animation Page 
- Profile page includes streaming URL
- Schema add url to DonationSetting for the streaming URL - ⚠️ remember to migrate!

### CODE REVIEW

File: src/components/DonationAnimation/\ [hunk 1]
```diff
@@ -0,0 +1,108 @@
+"use client";
```
Comment (hundehausen) [inline src/components/DonationAnimation/\]: This whole filename seems wrong

File: src/components/StreamingAddress.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        http://localhost:3000/animation/{url ?? "..."}
+      </div>
+      <p className="mb-4 select-none font-light leading-relaxed text-slate-600">
+        Paste this URL as a browser resource in your OBS studio
+      </p>
+    </div>
+  );
+}
+
+export default StreamingUrl;
```
Comment (hundehausen) [inline src/components/StreamingAddress.tsx]: Please export the same name as the filename

**Review by hundehausen — CHANGES_REQUESTED** [review]

File: src/components/DonationAnimation/\ (hunk 1 above)
Comment (AlexAnarcho) [reply src/components/DonationAnimation/\]: deleted - good catch!

File: src/components/StreamingAddress.tsx (hunk 2 above)
Comment (AlexAnarcho) [reply src/components/StreamingAddress.tsx]: renamed the file and according import to StreamingUrl as it is the more expressive name

**Review by hundehausen — APPROVED** [review]

**Pull request merged**